src/decaf.rs

```diff
@@ -194,6 +194,11 @@ impl<'de> Deserialize<'de> for DecafPoint {
 pub struct DecafPoint(pub ExtendedPoint);
 
 impl DecafPoint {
+    /// Convert this `DecafPoint` to its underlying `ExtendedPoint`.
```

Comment:
I don't think this conversion should exist, since the underlying point is an implementation detail.

Reply:
ack, yeah, removed in d39cb275